src/main/java/it/gov/pagopa/payhub/activities/activity/ingestionflow/receipt/ReceiptPagopaNotifySilActivityImpl.java

```diff
@@ -0,0 +1,24 @@
+package it.gov.pagopa.payhub.activities.activity.ingestionflow.receipt;
+
+import it.gov.pagopa.payhub.activities.dto.receipt.ReceiptNotifySilResult;
+import it.gov.pagopa.pu.debtposition.dto.generated.InstallmentDTO;
+import it.gov.pagopa.pu.debtposition.dto.generated.ReceiptDTO;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+/**
+ * Implementation of {@link ReceiptPagopaNotifySilActivity} for sending notification to SIL of received receipt.
+ * This class handles sending notification to SIL of received receipt.
+ */
+@Slf4j
+@Lazy
+@Component
+public class ReceiptPagopaNotifySilActivityImpl implements ReceiptPagopaNotifySilActivity {
+
+  @Override
+  public ReceiptNotifySilResult handleNotifySil(ReceiptDTO receiptDTO, InstallmentDTO installmentDTO) {
```

Comment:
I don't think we will have a result, it's ok or it's an exception
```suggestion
  public void handleNotifySil(ReceiptDTO receiptDTO, InstallmentDTO installmentDTO) {
```

Reply:
done